pyisic/_standards/ssic2020/ssic2020.py

```diff
@@ -0,0 +1,2226 @@
+# -*- coding: utf-8 -*-
+"""`SSIC2020 Standard <https://www.singstat.gov.sg/standards/standards-and-classifications/ssic>`_."""
+
+from ...types import Classification, Standard, Standards
+
+SSIC2020 = Standard(
+    standard=Standards.SSIC2020,
+    classes=[
+        Classification("A", "AGRICULTURE AND FISHING"),
+        Classification("01", "AGRICULTURE AND RELATED SERVICE ACTIVITIES"),
+        Classification("011", "GROWING OF CROPS, MARKET GARDENING AND HORTICULTURE"),
+        Classification("0111", "Growing of Food Crops (Non-Hydroponics)"),
+        Classification("01111", "Growing of leafy and fruit vegetables"),
```

Comment:
We should add categories here, look at [ISIC4](https://github.com/sayari-analytics/pyisic/blob/da74fc57e03a1b4f7aa318f9b99d83db0a184ce6/pyisic/_standards/isic4/isic4.py#L9) for example:
```suggestion
        Classification("A", "AGRICULTURE AND FISHING", Category.SECTION),
        Classification("01", "AGRICULTURE AND RELATED SERVICE ACTIVITIES", Category.DIVISION),
        Classification("011", "GROWING OF CROPS, MARKET GARDENING AND HORTICULTURE", Category.GROUP),
        Classification("0111", "Growing of Food Crops (Non-Hydroponics)", Category.CLASS),
        Classification("01111", "Growing of leafy and fruit vegetables", Category. SUBCLASS),
```
Look at using [regex find and replace in VSCode](https://stackoverflow.com/questions/43577528/visual-studio-code-search-and-replace-with-regular-expressions) to make this simpler.